config.json

```diff
@@ -566,7 +566,15 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "conditionals",
+        "loops",
+        "arrays",
+        "integers",
+        "enumerations",
```

Comment:
This is controversial, since `Enum` class was introduced in Python 3 - https://docs.python.org/3/library/enum.html
But I believe it's Ok to have it (to show that it's possible to use it)

Reply:
That is a good point, someone learning on Python 2.x wouldn't have access to enums without a 3rd party package. I removed it in the next commit, for consistency with the rest of the exercises (only one of which lists enumerations as a topic).

Reply:
@rtfoley probably it slipped through somehow, we should revisit it later